docs/modules/reference/pages/provisioning/detectors/BsfDetector.adoc

```diff
@@ -15,43 +15,46 @@ The BSF Detector passes the `map`, `ip_addr`, `svc_name`, and `results` variable
 
 == Configuration and use
 
-.Optional parameters for the BSF service detector
+.Optional parameters for the BSFDetector
 [options="header"]
 [cols="1,3a,1"]
 |===
 | Parameter
 | Description
-| Default
+| Default value
 
 | fileName
-| The path to the file on the OpenNMS server which contains the BSF-compatible script to run.
+| Path to file on OpenNMS server containing BSF-compatible script to run.
```

Comment:
```suggestion
| Path to the file on the OpenNMS server that contains the BSF-compatible script to run.
```
The updated version could be read as "to file" being the verb, so added back the "the". Similar with "server containing". Could be read that the path contains the BSF-compatible script rather than the OpenNMS server containing it.